subs.py

```diff
@@ -23,21 +23,26 @@
 
 
 def get_subtitles(video_id):
-    mediaelement_json = requests.get("http://psapi-granitt-prod-we.cloudapp.net/mediaelement/%s" % (video_id)).json()
-    if not mediaelement_json["hasSubtitles"]:
+    mediaelement_json = requests.get("https://psapi.nrk.no/playback/manifest/%s" % (video_id)).json()
+
+    if not mediaelement_json["playable"]['subtitles']:
         return None
-    ttml_sub_url = mediaelement_json["subtitlesUrlPath"]
-    html = requests.get(ttml_sub_url).text
-    if not html:
+    ttml_sub_url = mediaelement_json["playable"]['subtitles'][0]['webVtt']
+    subs = requests.get(ttml_sub_url).text
+    if not subs:
         return None
 
-    content = _ttml_to_srt(html)
+    content = vtt_to_srt(subs)
     filename = os.path.join(xbmc.translatePath("special://temp"), 'nor.srt')
     with open(filename, 'w' ,encoding='utf8') as f:
         f.write(content)
     return filename
 
 
+def vtt_to_srt(content):
```

Comment:
1. Is there no conversion from vtt to srt required?

2. Please make the the function private:
```suggestion
def _vtt_to_srt(content):
```

Reply:
Don't really know if there are any conversion needed. I'm not really familiar with the different subtitle-formats. 

I've been running this code for some time watching nrk content, and I've never noticed any issues with the subs.